revert of https://github.com/sbcl/sbcl/commit/4e2742958a22d0d651c02c92555dce9f0941d0a8
for some reason on ppc, this doesn't get rebuilt and fails the install
who knows why?
--
diff --git a/src/runtime/GNUmakefile b/src/runtime/GNUmakefile
index 6a62b2f..ffcc3d2 100644
--- a/src/runtime/GNUmakefile
+++ b/src/runtime/GNUmakefile
@@ -178,7 +178,7 @@ TAGS tags: $(SRCS) $(HEADERS) $(INC)
 	@etags --language=c $(SRCS) $(HEADERS) $(INC) || true
 
 clean:
-	-rm -f *.[do] ../../tlsf-bsd/tlsf/tlsf.o $(TARGET) *.tmp libsbcl.a sbcl.h \
+	-rm -f *.[do] $(TARGET) *.tmp libsbcl.a sbcl.h \
  ldb unit-tests libsbcl.a shrinkwrap-sbcl* sbcl.mk core
 
 %.d: %.c sbcl.h
